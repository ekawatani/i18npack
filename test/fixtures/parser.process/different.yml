foo: foo_text
bar: bar_text
baz: "{{ foo }} and {{ bar }}"