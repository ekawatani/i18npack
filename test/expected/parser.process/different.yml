foo: foo_text
bar: bar_text
baz: foo_text and bar_text